tests/data/AnsysSound_FRF_bad_2024R2_20241206.txt

```diff
@@ -0,0 +1,29 @@
+AnsysSound_WRONGHEADER 1
```

Comment:
same comment as previous one: @ansaminard we should we use a realistic FRF here (this file looks like a spectrum)

Reply:
@anssmolla That's the point! Look at how this file used: it is used to test that an error is produced